packages/checkout/components/text-input/validated-text-input.tsx

```diff
@@ -122,7 +133,11 @@ const ValidatedTextInput = ( {
 		};
 	}, [ clearValidationError, errorIdString ] );
 
-	if ( isString( passedErrorMessage ) && passedErrorMessage !== '' ) {
+	if (
+		isString( passedErrorMessage ) &&
+		passedErrorMessage !== '' &&
+		isObject( passedErrorMessage )
```

Comment:
Previously, if `validationError` was not an object, this would error because you can't access `.message` on undefined.